components/admin_console/admin_definition.jsx

```diff
@@ -3825,8 +3857,11 @@ const AdminDefinition = {
                         label: t('admin.saml.enableAdminAttrTitle'),
                         label_default: 'Enable Admin Attribute:',
                         isDisabled: it.any(
+
+                            // Should this be removed?
                             it.not(it.isSystemAdmin),
```

Comment:
No this attribute can make people a system admin allowing them to manipulate more than what they have control too. Leave it in.